src/cata_tiles.h

```diff
@@ -481,15 +480,11 @@ class cata_tiles
         tripoint zone_offset;
 
         // offset values, in tile coordinates, not pixels
-        int o_x = 0;
-        int o_y = 0;
+        point o = point_zero;
```

Comment:
`point` and `tripoint` are automatically initialized to all components being 0. `sprite_offset` does not have this explicit initialization. For consistency, it should not be used for the other points either.

Reply:
I was on the fence about this and went for the option I did on the basis of "porting the spirit of the code".  But I think you're right; better to do without.